Functions.php

```diff
@@ -43,6 +42,7 @@ public static function register($pdo)
      * @param string $string column value
      * @param string $separator separator added between values
      */
+    // phpcs:ignore PSR1.Methods.CamelCapsMethodName.NotCamelCaps
```

Comment:
any reason why we can't rename this method to `groupConcatStep`?

Reply:
No good reasons, except that I generally avoid renaming public methods. But those two are safe, so I've just renamed them.